App/Containers/Contact.tsx

```diff
@@ -101,6 +104,13 @@ class ContactModal extends React.Component<Props> {
     }
   }
 
+  constructor(props: Props) {
```

Comment:
i've never been sure the difference between setting the state in a constructor versus just setting it in the root tree... neither our lint nor compiler have a preference, so neither do I, yet.

Reply:
Hmm I'm not sure, I've always set it in the constructor just because that's what the React docs do.

Reply:
👍